clients/ui/frontend/src/app/pages/modelRegistry/ModelRegistryRoutes.tsx

```diff
@@ -16,6 +18,18 @@ const ModelRegistryRoutes: React.FC = () => (
       }
     >
       <Route index element={<ModelRegistry empty={false} />} />
+      <Route path="registeredModels/:registeredModelId">
+        <Route index element={<Navigate to={ModelVersionsTab.VERSIONS} replace />} />
+        <Route
+          path={ModelVersionsTab.VERSIONS}
+          element={<ModelVersions tab={ModelVersionsTab.VERSIONS} empty={false} />}
+        />
+        <Route
+          path={ModelVersionsTab.DETAILS}
+          element={<ModelVersions tab={ModelVersionsTab.DETAILS} empty={false} />}
+        />
```

Comment:
Just to disclose the private conversation, there will be a follow up to add the archive view here.